charts/rancher-turtles/values.yaml

```diff
@@ -3,3 +3,5 @@ rancherTurtles:
   tag: v0.0.0
   imagePullPolicy: Never
   namespace: rancher-turtles-system
+cluster-api-operator:
+  enabled: true
```

Comment:
Should this also include `create-namespace` and `namespace` flags? So it won’t be installed in the default namespace.

Reply:
it gets the same namespace as rancher turtles by default, which I guess is ok for us